openslides/motions/views.py

```diff
@@ -618,6 +618,8 @@ def numbering(self, request, pk=None):
 
                 # Set new identifers and change identifiers of amendments.
                 for obj in motions_to_be_sorted:
+                    if Motion.objects.filter(identifier = obj['new_identifier']).exists():
```

Comment:
Please remove the whitespaces around the `=`.

Reply:
(The only travis error)

Reply:
done